pkgs/applications/audio/argotlunar/default.nix

```diff
@@ -0,0 +1,48 @@
+ { pkgs } :
+
+pkgs.stdenv.mkDerivation rec {
```

Comment:
And following that first comment: this should be straight up `stdenv.mkDerivation` instead of `pkgs.stdenv.mkDerivation` (same with all your `pkgs.` references in here).